### ISSUE TITLE
Move the model endpoints (ModelBean) to a microservice

### ISSUE THREAD

**Issue by ivakegg** [NationalSecurityAgency/datawave#1454]
The ModelBean endpoints need to be moved to a microservice.  Initial thoughts are that this could be moved into the existing data dictionary endpoint, but can be separated if needed.  In the end the ModelBean should go directly to the microservice for backward compatibility (see the DataDictionaryBean as an example).

**Comment by ivakegg** [comment]
You can use the feature/queryMicroservices datawave branch as your base, and the feature/spring-boot-2.4 datawave-dictionary-service branch as the microservice destination.

**Comment by jwomeara** [comment]
yes... feature/spring-boot-2.4.... which is running spring boot 2.6.3...  d'oh!
